src/Build.UnitTests/BackEnd/SdkResolverLoader_Tests.cs

```diff
@@ -137,7 +137,7 @@ public void VerifyThrowsWhenResolverHasNoPublicConstructor()
                 sdkResolverLoader.LoadResolvers(_loggingContext, ElementLocation.EmptyLocation);
             });
 
-            exception.Message.ShouldBe($"The SDK resolver type \"{nameof(MockSdkResolverNoPublicConstructor)}\" failed to load. No parameterless constructor defined for this object.");
+            Assert.True(exception.Message.StartsWith($"The SDK resolver type \"{nameof(MockSdkResolverNoPublicConstructor)}\" failed to load."));
```

Comment:
Shouldly has a `ShouldStartWith` on string.

Reply:
Ah, thanks. I couldn't find it in their documentation. Updating.

Reply:
Updated. `git rebase -i` FTW!